packages/rspack/src/stats.ts

```diff
@@ -7,7 +7,7 @@
  * Copyright (c) JS Foundation and other contributors
  * https://github.com/webpack/webpack/blob/main/LICENSE
  */
-import * as binding from "@rspack/binding";
+import type binding from "@rspack/binding";
```

Comment:
`type * as binding`

Reply:
Everywhere else should be the same like this too.

Reply:
thank